include/clap/ext/draft/context-menu.h

```diff
@@ -0,0 +1,121 @@
+#pragma once
+
+#include "../../plugin.h"
+
+// This extension lets the host and plugin exchange menu items and let the plugin ask the host to
+// show its context menu.
+
+static CLAP_CONSTEXPR const char CLAP_EXT_CONTEXT_MENU[] = "clap.context-menu.draft/0";
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// There can be different target kind for a context menu
+enum {
+   CLAP_CONTEXT_MENU_TARGET_KIND_GLOBAL,
+   CLAP_CONTEXT_MENU_TARGET_KIND_PARAM,
+};
+
+// Describes the context menu target
+typedef struct clap_context_menu_target
+{
+   uint32_t kind;
```

Comment:
Maybe better call it 'type' ?

Reply:
@baconpaul you're native speaker, what fits better here type or kind?

Reply:
English language either works. Have we made a tyoe / kind choice elsewhere in extensions? That’s what I would look for

Reply:
Type is a reserved keyword in many languages, but CLAP already uses the word in other places as well. (IIRC the Rust bindings use either `type_` or `typ` for those fields/params, don't recall which one is used)

Reply:
Then better to continue with `kind` I suppose.